protobuf-provider/src/main/java/io/confluent/kafka/schemaregistry/protobuf/ProtobufSchema.java

```diff
@@ -2289,27 +2289,22 @@ public Object transformMessage(RuleContext ctx, FieldTransform transform, Object
 
   private Object toTransformedMessage(
       RuleContext ctx, Descriptor desc, Object message, FieldTransform transform) {
-    if (desc == null || message == null) {
+    FieldContext fieldCtx = ctx.currentField();
```

Comment:
What happen if `message == null`? Do we need to fail early?

Reply:
If a value is a field, we allow a null value to be transformed to a non-null value.